aws/resource_aws_macie2_classification_job_test.go

```diff
@@ -0,0 +1,307 @@
+package aws
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/naming"
+)
+
+func TestAccAwsMacie2ClassificationJob_Name_Generated(t *testing.T) {
+	var macie2Output macie2.DescribeClassificationJobOutput
+	resourceName := "aws_macie2_classification_job.test"
+	bucketName := os.Getenv("AWS_S3_BUCKET_NAME")
+	accountID := os.Getenv("AWS_ACCOUNT_ID")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      nil,
```

Comment:
This should have a `CheckDestroy` function that confirms that the job has been `CANCELLED`